frontend/src/components/Banner.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+interface BannerProps {
+  title?: string;
+}
+
+const Banner: React.FC<BannerProps> = ({
+  title = "System for Operational Logistics Orders (SOLO)"
+}) => <h1>{title}</h1>;
+
+export default Banner;
```

Comment:
Not a huge issue either way but preference of named or default exports? Worth being consistent but either is fine.

Reply:
I think naming the component like `const Banner = ....`, and then exporting that as default gives the best experience in React dev tools but I'm fine with either.